scale-ui/app/modules/jobs/partials/jobDetailTemplate.html

```diff
@@ -27,11 +27,20 @@ <h4 class="panel-title error-title"><strong>{{ vm.job.error.category }} Error: {
                         {{ vm.job.error.description }}
                     </div>
                 </div>
+                <div class="panel panel-info" ng-show="vm.job.is_superseded">
+                    <div class="panel-heading">
+                        <div class="panel-title">
+                            <strong>SUPERSEDED {{ vm.supersededChange }}</strong>
+                        </div>
+                    </div>
+                </div>
```

Comment:
I like this a lot. Not sure if you saw my comment somewhere, but if there is no superseded by job then it should just say SUPERSEDED with the timestamp.

Yeah, I haven't been sold on any color yet. We could leave it for now and change it later. Maybe we'll get a good suggestion at review time.